lib/install/management/store_files.go

```diff
@@ -88,13 +88,13 @@ func (d *Dispatcher) deleteImages(conf *config.VirtualContainerHostConfigSpec, v
 			continue
 		}
 
-		if len(children) == 1 { // when image store path is given, we need to delete VIC dir too when it is empty.
+		if len(children) == 1 || len(children) == 0 { // when image store path is given, we need to delete VIC dir too when it is empty.
```

Comment:
Not familiar with these code. What does `|| len(children) == 0 ` mean?

Reply:
i think it is empty dir

Reply:
> i think it is empty dir

So in vc7.0, `len(children) == 0 or 1`  means empty dir, while in vc6.x only `len(children) == 1` means empty dir